pandas/_config/config.py

```diff
@@ -461,16 +461,17 @@ def register_option(key: str, defval: object, doc="", validator=None, cb=None):
             raise ValueError(f"{k} is a python keyword")
 
     cursor = _global_config
-    msg = "Path prefix to option '{option}' is already an option"
     for i, p in enumerate(path[:-1]):
         if not isinstance(cursor, dict):
-            raise OptionError(msg.format(option=".".join(path[:i])))
+            option = ".".join(path[:i])
+            raise OptionError(f"Path prefix to option '{option}' is already an option")
         if p not in cursor:
             cursor[p] = {}
         cursor = cursor[p]
 
     if not isinstance(cursor, dict):
-        raise OptionError(msg.format(option=".".join(path[:-1])))
+        option = ".".join(path[:-1])
+        raise OptionError(f"Path prefix to option '{option}' is already an option")
```

Comment:
@WillAyd @datapythonista : I'm somewhat ambivalent about this change actually.  As much as I appreciate the convenience of f-strings, the fact we have to duplicate the error message gives me pause.

Thoughts?

Reply:
Yea valid point. Probably better use case for .format